project/plugins.sbt

```diff
@@ -1 +1,2 @@
 addSbtPlugin("org.apache.daffodil" % "sbt-daffodil" % "1.1.0")
+addSbtPlugin("com.owlcyberdefense" % "sbt-flowerpress" % "1.0.0")
```

Comment:
This is a public schema. We should remove this plugin since it is not available to the public.